tools/releaser/scripts/breaking-changes.sh

```diff
@@ -1,37 +1,45 @@
 #!/bin/bash
 set -eu
-GOPATH=$(go env GOPATH)
 
 # Inputs:
-# API_DIFF_OLD_COMMIT: commit before the API changes to compare with. If not provided, script will fail with "unbound variable" error
-# API_DIFF_NEW_COMMIT: commit with the new API changes. If not provided, script will fail with "unbound variable" error
-# TARGET_BREAKING_CHANGES_FILE - file to save breaking changes
 TARGET_BREAKING_CHANGES_FILE=${TARGET_BREAKING_CHANGES_FILE:-""}
 script_path=$(dirname "$0")
 
-echo "Installing go-apidiff"
-go install github.com/joelanford/go-apidiff@latest > /dev/null
+# shellcheck source=/dev/null
+source "$script_path/extract-version.sh"
+BASE_VERSION="github.com/mongodb/atlas-sdk-go/$SDK_MAJOR_VERSION@$SDK_VERSION"
 
-echo "Running breaking changes check comparing commits ${API_DIFF_OLD_COMMIT} and ${API_DIFF_NEW_COMMIT}"
+echo "Installing gorelease"
+go install golang.org/x/exp/cmd/gorelease@latest >/dev/null
```

Comment:
:FYI: Latest golang does have tools directive that allows us to define tools direclty in the go.mod.
See: https://github.com/golang/go/issues/48429#issuecomment-2547760589